package.json

```diff
@@ -48,34 +48,34 @@
     "iterall": "^1.2.2"
   },
   "devDependencies": {
-    "@babel/cli": "^7.0.0-beta.51",
-    "@babel/core": "^7.0.0-beta.51",
-    "@babel/node": "^7.0.0-beta.51",
-    "@babel/plugin-proposal-class-properties": "^7.0.0-beta.51",
-    "@babel/plugin-proposal-object-rest-spread": "^7.0.0-beta.51",
-    "@babel/plugin-syntax-async-generators": "^7.0.0-beta.51",
-    "@babel/plugin-transform-classes": "^7.0.0-beta.51",
-    "@babel/plugin-transform-destructuring": "^7.0.0-beta.51",
-    "@babel/plugin-transform-flow-strip-types": "^7.0.0-beta.51",
-    "@babel/plugin-transform-modules-commonjs": "^7.0.0-beta.51",
-    "@babel/plugin-transform-spread": "^7.0.0-beta.51",
-    "@babel/polyfill": "^7.0.0-beta.51",
-    "@babel/preset-env": "^7.0.0-beta.51",
-    "@babel/register": "^7.0.0-beta.51",
-    "babel-eslint": "8.2.4",
+    "@babel/cli": "7.0.0-beta.53",
```

Comment:
Even though we use `yarn` I think it's better to use explicit versions in `package.json`.

Reply:
For devDependencies you're probably right. That's definitely not true for regular dependencies, as it can cause people to needlessly download multiple compatible versions of the same library.